Prox/Prox/Utilities/Eventbrite.swift

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Alamofire
+import Deferred
+
+private let defaultRadiusKm = 2
+
+private let baseURL = "https://www.eventbriteapi.com/v3/"
+private let eventSearchURL = baseURL + "events/search/"
+
+struct Eventbrite {
+
+    private static let oAuthToken: String = {
+        return APIKey.EVENTBRITE_ANONYMOUS_OAUTH.get()
+    }()
+
+    private static let defaultHeaders = [
+            "Authorization": "Bearer \(oAuthToken)",
+            "Accept": "application/json",
+    ]
+
+    static func searchEvents(near location: CLLocationCoordinate2D, withRadiusKm radiusKm: Int = defaultRadiusKm, keyword: String? = nil, sortType: EventbriteSortType = .Distance) -> Future<DatabaseResult<[Event]>> {
+        let params = [
+            "q": keyword,
+            "sort_by": sortType.name,
+            "location.latitude": String(location.latitude),
+            "location.longitude": String(location.longitude),
+            "location.within": "\(radiusKm)km",
+            "start_date.keyword": "this_week", // TODO: choose date range. as arg?
+            "expand": "logo,venue,format,category", // todo: choose. can slows down requests.
```

Comment:
Clarify comment (grammatical?) and maybe add a link to the expansions part of the doc (I was only looking at the events/search and didn't realize what expand meant.